Projects/Shapes/Shapes/Shapes/StaticArray.h

```diff
@@ -0,0 +1,92 @@
+#pragma once
+
+template<typename T, int N>
+class TStaticArray
+{
+private:
+	int Size = N;
```

Comment:
It may be worth it to have this variable be const or even const static since it will never change